apps/sms/views/shared/js/utils.js

```diff
@@ -963,6 +963,17 @@
       });
 
       return defer.promise;
+    },
+
+    /**
+     * Check if device have more than 1 active SIM.
+     * @returns {boolean} True if the device has more than 1 SIM port and at
+     *  least 2 SIMs are inserted and false if device only has 1 SIM port or
+     *  mozIccManager not supported.
+     */
+    hasSeveralSim() {
```

Comment:
IMO ```Utils``` is better place for this method \0/ I see you replaced ```Settings.hasSeveralSim``` in several places, but there are still two places where old ```Settings.hasSeveralSim``` is used (message manager and messaging service), do you think we can entirely migrate to ```Utils.hasSeveralSim```  and get rid of settings related method/tests in this patch or you'd rather prefer to change it in a follow up?